interface/client/appStart.js

```diff
@@ -35,17 +29,20 @@ mistInit = function(){
             });
         }
 
-
         Tabs.upsert({_id: 'wallet'}, {
             url: 'https://wallet.ethereum.org',
             position: 1,
             permissions: {
                 admin: true
             }
-        });        
-    })
-    .then(function() {
-        window.trigger('mist-ready');
```

Comment:
Yes, this was also bugging me and giving an error. @hiddentao do you remember what it was for?